### DISCUSSION TITLE
Is it possible to have tqdm print to a new line each time?

### DISCUSSION THREAD

**Discussion by N-Demir** [tqdm/tqdm#1498] [Q&A]
Spent a few hours trying to find the answer in the issues / discussions but I just can't seem to figure it out. I would like the tqdm to print each update on a new line so that eventually I can have my script send stdout/stderror to a log file and have that be ingested by cloudwatch etc.

Looking around, it seems like this should work? But when testing locally and running this script it still overwrites the progress bar with each update. 
```
from time import sleep

from tqdm import tqdm

if __name__ == "__main__":
    for i in tqdm(range(9), position=-1):
        sleep(1)
```

Both #966 and #1319 make it seem like this should work. What am I missing?

**Comment by hetari** [comment]
try this:
```py
from time import sleep

from tqdm import tqdm

if __name__ == "__main__":
    for _ in tqdm(range(9), ncols=100):
        sleep(1)
```

**Comment by ovianao** [comment]
If you have a description, you can also prepend it with newline, and it seems to work.
Though I agree, that it would be ideal to have an official option that supports this.

Use Case:  non-tty devices in some cases (looking at you, Jenkins), don't update the console unless a newline is printed.  This leads to unwanted behaviour since it can monitor the console for stuck jobs, and long running processes that rely on tqdm to update on progress don't update that console unless a newline gets sent.

```
from tqdm import tqdm
import time

for i in tqdm(range(10), desc="\nname"):
    time.sleep(1)
```
